buzsaki_lab_to_nwb/valero/eventsinterface.py

```diff
@@ -48,7 +48,7 @@ def run_conversion(self, nwbfile: NWBFile, metadata: dict, stub_test: bool = Fal
 
         # Create TimeIntervals
         name = "UpDownStatesTimeIntervals"
-        description = "TBD"
+        description = "TBD"  # TODO Cody, what do you think is a good description of this?
```

Comment:
Looking for your opinion here @CodyCBakerPhD .

Reply:
lol

Y'know, I really have no clue. It's a very specific scientific classification used by this lab (not sure if I've seen it used anywhere else...)

I normally do the 'prettier placeholder' for this; `"Classifications of UP/DOWN neural states."`

As for the actual description of what those are, what they mean, how they are classified - I'd recommend asking the author or deferring to the publication

Reply:
Fine to add in follow-up if you want